index.html

```diff
@@ -1,10 +1,138 @@
+        <li>Настройка рабочей среды</li>
+        <li>Краткая история HTML</li>
+        <li>Типы тегов и атрибуты</li>
+        <li>Теги для работы с текстом</li>
+        <li>Создание собственной страницы</li>
+        <li>Основы ветвления в Git</li>
+        <li>Создание pull request на GitHub</li>
+      </ul>
+      </li>
+      <li>Погружение в HTML-разметку
+        <ul>
+        <li>Мета-теги</li>
+        <li>Мнемоники</li>
+        <li>Теги для работы со списками</li>
+        <li>Теги для работы с таблицами</li>
+        <li>Теги для работы с формами</li>
+        <li>div и span</li>
+      </ul>
+      </li>
+    </ol>
+    <table border="1" width="50%">
+      <thead>
+      <tr>
+        <th>Tag</th>
+        <th>Description</th>
+      </tr>
+      </thead>
+      <tbody>
+      <tr>
+        <td>&#60table&#62</td>
+        <td>Defines a table</td>
+      </tr>
+      <tr>
+        <td>&#60tr&#62</td>
+        <td>Defines a row in a table</td>
+      </tr>
+      <tr>
+        <td>&#60th&#62</td>
+        <td rowspan="2">Defines a &#34header cell&#34 &#8260 &#34cell&#34 in a table</td>
+      </tr>
+      <tr>
+        <td>&#60td&#62</td>
+      </tr>
+      <tr>
+        <td colspan="2">Rarely used</td>
+      </tr>
+      <tr>
+        <td>&#60caption&#62</td>
+        <td>Defines a table caption</td>
+      </tr>
+      <tr>
+        <td>&#60thead&#62</td>
+        <td>Groups the header content in a table</td>
+      </tr>
+      <tr>
+        <td>&#60tbody&#62</td>
+        <td>Groups the body content in a table</td>
+      </tr>
+      <tr>
+        <td>&#60tfoot&#62</td>
+        <td>Groups the foot content in a table</td>
+      </tr>
+      </tbody>
+<tfoot>
+</tfoot>
+    </table>
+    <p></p>
+    <form action="/" method="get">
+      <fieldset>
+        <legend>My first form</legend>
+        <label for="name">Name</label> <br>
+        <input id="name"  type="text" name="Name" placeholder="Name" required/><br>
+        <label for="email">Email</label> <br>
+        <input id="email"  type="email" name="Email" required/> <br>
+        <label for="password">Password</label> <br>
+        <input id="password"  type="password" name="password" required/> <br>
+        <label for="level">Your programming level</label> <br>
+        <input id="level"  type="range" name="level" min="0" max="10" />
+        <p>
+          What does  HTML stand for&#63
+        </p>
+        <input id="html-1" type="checkbox" name="html" value="off" />
+        <label for="html-1">Hyperlinks and Text Markup Language</label> <br>
+        <input id="html-2" type="checkbox" name="html" value="off" />
+        <label for="html-2">Home Tool Markup Language</label> <br>
+        <input id="html-3" type="checkbox" name="html" value="on"/>
+        <label for="html-3">Hyper Text Markup Language</label> <br>
+        <p>
+          What does  HTTP stand for&#63
+        </p>
+        <input id="http-1" type="checkbox" name="http" value="off" />
```

Comment:
Look at the task carefully and change the inputs in this block.

Reply:
Changed